docs/docsite/rst/user_guide/playbooks_delegation.rst

```diff
@@ -77,6 +77,33 @@ You can also mix and match the values::
 .. note::
      No matter how small the percentage, the number of hosts per pass will always be 1 or greater.
 
+You can also ignore unreachable node to go ahead with your job, suppose for example you have 6 nodes and you
+want to divide it in groups of 2, an example of serialization option should be::
+
+    ---
+
+    - hosts: webservers
+      serial: [[2, 1], 2, 2]
+
+
+In this case, the first group represented by the list [2, 1] has a value of 1 (True) regarding the question
+"Do you want to ignroe unreachable?". For others groups the implicit answer to the question is 0 (False).
+At this point if one or both machines in the first group of 2 are unreachable, ansible go ahead with the
+second group and does not stop the play. If in the second group one machine is unreachable, ansible does not
+stop (this is the default behavoir also in old versions of ansible). If in the second group both machines
+are unreachable, ansible stop the play.
+
+.. note::
+   Valid value are only 0 and 1, False and True, other values will be ignored and 0 will be used. Example::
+
+   ---
+
+    - hosts: webservers
+      serial: [[2, 1], [2,0], 2, [4, False], [3, True], [2, "False"]]
+
+
+Last value will result as [2, 0] because "False" in this case is a string and not a boolean.
+
```

Comment:
I would like to modify the note to

.. note::
   Valid values are only 0 (False) and 1 (True), all other values will be ignored and 0 will be used. Example::

   ---

    - hosts: webservers
      serial: [[2, 1], [2, 2], [2, "one"], [2,0], 2, [4, False], [3, True], [2, "False"]]


The second list [2, 2] is not valid and ansible will replace it with [2, 0]
The third list [2, "one"] is not valid and ansible will replace it with [2, 0]
In the fifth position we have a single value (2) that implicitly represents [2, 0]
The last list, [2, "False"] is not valid, because "False" in this case is a string and not a boolean and ansible will replace it with [2, 0]